public/src/components/channelManagement/headTests/headTestNewVariantButton.tsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
```

Comment:
this file isn't used

Reply:
That's interesting! Checking through, I can see that `testNewVariantButton.tsx` is a copy of`banner/bannerTestNewVariantButton.tsx` (only file paths adjusted for folder nesting) which is possibly why I didn't catch this. Happy to delete `head/headTestNewVariantButton.tsx`, but if someone deletes `testNewVariantButton.tsx` in the future then that would break our new header tests page display.

Reply:
Just checked this - `banner/bannerTestNewVariantButton.tsx` is also not used and can be deleted!
`testNewVariantButton.tsx` (used by `testVariantsEditor.tsx`) is used by epic/banner/header.
We have tried to share as much as possible between the epic + banner tools, and the header should also.

Reply:
> but if someone deletes testNewVariantButton.tsx in the future then that would break our new header tests page display.
not sure why anyone would do this, but TS would catch it!

Reply:
Will delete